app/controllers/courses_controller.rb

```diff
@@ -42,6 +42,22 @@ def description
   end
 
   def about
+    if params[:display_errors].present?
+      attributes = %w[about_course interview_process how_school_placements_work]
+
+      errors = @course.publish(provider_code: @provider.provider_code).errors
```

Comment:
If I shared a link with `display_errors` in the URL, but the errors had been fixed, would this accidentally publish the course?

Reply:
Yes, good point, which is why we should do a custom endpoint instead...

Reply:
No, because we hit the publish endpoint which would return no errors if there are no errors